spec/facades/user_campaigns_with_characters_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require_relative '../../app/facades/user_campaigns_with_characters'
+
+describe 'UserCampaignsWithCharacters' do
+
+  it 'Should have respond to user' do
+    character = FactoryGirl.create :character
```

Comment:
Use `FactoryGirl` short syntax instead. You'll need to add the following line on `rails_helper` file

```ruby
config.include FactoryGirl::Syntax::Methods
```